src/exportDataAction/ExportDataAction.js

```diff
@@ -66,8 +69,27 @@ class ExportDataAction {
     }
   }
 
+  historicalFilterString() {
+    const sessionFilter = this.sessionService.getHistoricalSessionFilter();
+
+    if (!sessionFilter) {
+      return '';
```

Comment:
If you are checking for presence of a truthy sessionFilter in `runExportTask` later, you can change this to just return a value if sessionFilter is truthy.